collections/_article/dev-snapshot-godot-4-5-beta-2.md

```diff
@@ -0,0 +1,84 @@
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.5.beta2/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+---
+
+*The cover illustration is from* [**Xion Leak**](https://store.steampowered.com/app/1948490/Xion_Leak/?curator_clanid=41324400), *a fast-paced, co-op platformer where you win trophies by evading the dangers of the mysterious Foreman! You can buy the game [on Steam](https://store.steampowered.com/app/1948490/Xion_Leak/?curator_clanid=41324400), and follow follow the developers [on Bluesky](https://bsky.app/profile/gentlebeastsstudio.bsky.social).*
+
+## Highlights
+
+For an overview of what's new overall in Godot 4.5, have a look at the highlights for [4.5 beta 1](/article/dev-snapshot-godot-4-5-beta-1/), which cover a lot of the changes. This blog post only covers the changes between beta 1 and beta 2. This section covers the most relevant changes made since the beta 1 snapshot, which are largely regression fixes:
+
+### Input: SDL3 joystick input driver
+
+Even though we're now in feature freeze, you might recall us mentioning a handful of pre-approved exceptions. This was by far the biggest one of note: **SDL3 input**! For those unaware: <abbr title="Simple DirectMedia Layer">SDL</abbr> is a collection of common APIs shared across an ever-growing portion of software, allowing for convenient and easy integration of common tasks and actions for developers. They've recently came out with their 3.0 release [this January](https://www.patreon.com/posts/120491416), bringing with it a multitude of features and enhancements alike.
+
+First-time contributior [Nintorch](https://github.com/Nintorch) took to the integration of one such feature: their input handler ([GH-106218](https://github.com/godotengine/godot/pull/106218)). So thorough was this implementation, that it actually entirely *supplanted* our old input method! While the more technical nuances of this change are better delved into on the PR itself, the biggest takeaway is that this opens the door for easy & native integration of additional input features; while we won't be seeing any of these in 4.5, Nintorch already has a [work-in-progress PR](https://github.com/godotengine/godot/pull/107967) that's set to add these features.
+
+### And more!
+
+There are too many exciting changes to list them all here, but here's a curated selection:
```

Comment:
They're certainly "exciting" for us, but a lot of this is very technical mumbo-jumbo. Sugar-coating it a bit too much perhaps?